augur/measurements.py

```diff
@@ -0,0 +1,364 @@
+             "field in the config JSON will be dropped.")
+    config.add_argument("--key",
+        help="A short key name of the collection for internal use within auspice." +
+             "If not provided via config or command line option, the collection TSV filename will be used. ")
+    config.add_argument("--title",
+        help="The full title of the collection to display in the measurements panel title. " +
+             "If not provided via config or command line option, the panel's default title is 'Measurements'.")
+    config.add_argument("--x-axis-label",
+        help="The short label to display for the x-axis that describles the value of the measurements. " +
+             "If not provided via config or command line option, the panel's default x-axis label is 'measurements values'.")
+    config.add_argument("--threshold", type=float,
+        help="A measurements value threshold to be displayed as a single grey line shared across subplots.")
+    config.add_argument("--filters", nargs="+",
+        help="The columns that are to be used a filters for measurements. " +
+             "If not provided, all columns will be available as filters.")
+    config.add_argument("--group-by", type=str,
+        help="The default grouping column. If not provided, the first grouping will be used.")
+    config.add_argument("--measurements-display", type=str, choices=["raw", "mean"],
+        help="The default display of the measurements")
+
+    config.add_argument("--show-overall-mean", "--hide-overall-mean",
+        dest="show_overall_mean", action=HideAsFalseAction, nargs=0,
+        help="Show or hide the overall mean per group by default")
+    config.add_argument("--show-threshold", "--hide-threshold",
+        dest="show_threshold", action=HideAsFalseAction, nargs=0,
+        help="Show or hide the threshold by default. This will be ignored if no threshold is provided.")
+
+    optional_settings = export.add_argument_group(
+        title="OPTIONAL SETTINGS"
+    )
+    optional_settings.add_argument("--minify-json", action="store_true",
+        help="Export JSON without indentation or line returns.")
+
+
+    concat = subparsers.add_parser("concat", help="Concatenate multiple measurements JSONs into a single JSON file")
+    concat.add_argument("--jsons", required=True, type=str, nargs="+", metavar="JSONs",
+        help="Measurement JSON files to concatenate.")
+    concat.add_argument("--default-collection", type=str,
+        help="The key of the default collection to display. " +
+             "If not provided, the first collection of the first JSON file will be displayed")
+    concat.add_argument("--minify-json", action="store_true",
+        help="Concat JSONs without indentation or line returns.")
+    concat.add_argument("--output-json", required=True, metavar="JSON", type=str,
+        help="Output JSON file")
+
+
+def run(args):
+    if args.subcommand == 'export':
+        return export_measurements(vars(args))
```

Comment:
I used `vars(args)` because it's used in the validate module. I see now that it's being used as `**vars(args)` [in the validate module](https://github.com/nextstrain/augur/blob/8014186d35c13d9f131cfa4828c6b7f81932909f/augur/validate.py#L147) to pass args as keyword arguments to the subcommand functions. 

I can change to passing the `args` object directly to fit in with other Augur subcommands.

Reply:
Addressed in https://github.com/nextstrain/augur/pull/879/commits/c8a2e0ae82625864eb527c5c6db5dfea4b3d4666